editor/webpack.config.js

```diff
@@ -166,6 +166,7 @@ const config = {
       'process.env.JEST_WORKER_ID': 'undefined',
       'process.env.HOT_MODE': hot,
       'process.env.HMR': false,
+      'process.env.REMIX_DEV_ORIGIN': "'http://localhost:8000'",
```

Comment:
Can you use an `isDev` check here please, so that we won't accidentally end up pointing at localhost if we ever switch to using a dev mode of remix in the deployed editor

Reply:
Fixed in f7b48c4a8.